### ISSUE TITLE
twisted.test.test_process.Win32ProcessTestCase.testBadArgs fails on win32er and iocpr

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1800]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1800|
|Type|defect|
|Created|2006-06-07 01:11:47Z|

It raises a TypeError on one and a different TypeError on the other.

<details><summary>Searchable metadata</summary>

```
trac-id__1800 1800
type__defect defect
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1149642707000000 1149642707000000
changetime__1474197555542286 1474197555542286
version__None None
owner__ghazel ghazel

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @ghazel|
|-|-|

I checked some changes in to win32-proc-1800.  I didn't run the tests because I don't have a win32 machine around.  The changes are pretty simple though, so maybe I got it right.

**Comment by twisted-trac** [comment]
|[<img alt="ghazel's avatar" src="https://avatars.githubusercontent.com/u/31402?s=50" width="50" height="50">](https://github.com/ghazel)|@ghazel commented|
|-|-|

Works for me, but I didn't run buildbot on it. Where are the instructions for that?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [17108]) Merge win32-proc-1800

Author: exarkun
Reviewer: alus
Fixes [#1800](https://github.com/twisted/twisted/issues/1800)

Reject unsupported arguments in a manner consistent with that of the other
reactors.  Fixes the testBadArgs process test on win32er and iocpr.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
